src/components/desktop/desktop.vue

```diff
@@ -16,15 +25,20 @@
             </LocationProvider>
           </div>
           <LocationProvider location="predictive_layer">
-            <RelatedTags v-if="!$x.isEmpathizeOpen && $x.relatedTags.length > 0" />
+            <RelatedTags v-if="$x.relatedTags.length > 0" />
```

Comment:
Style the sliding pannel button: 
![image](https://user-images.githubusercontent.com/5759712/175527325-e08ae6b8-02b0-4d74-8fc5-cf3b84893c0e.png)